elements/rh-footer/demo/rh-footer.html

```diff
@@ -93,3 +75,27 @@ <h3 slot="links-secondary" data-analytics-text="Red Hat legal and privacy links"
     </ul>
   </rh-footer-universal>
 </rh-footer>
+
+<link rel="stylesheet" href="https://static.redhat.com/libs/redhat/redhat-font/4/webfonts/red-hat-font.min.css">
+<link rel="stylesheet" href="https://static.redhat.com/libs/redhat/redhat-theme/6/advanced-theme.css">
```

Comment:
@heyMP what is the dependency for footer on the advance-theme.css?   

@bennypowers do we not load the red-hat-fonts now in the demos/docs now? I believe when these demos were created we didn't.

Might be able to remove these

Reply:
Also I don't perceive a difference when I remove these files form the demo, but just want to be sure before commiting it as a change.

Reply:
We could remove them, just make sure they exist in all of the three existing demofile targets:
1. Dev server template head
2. 11ty demo files head
3. Playground hidden html